packages/react-scripts/config/jest/fileTransform.js

```diff
@@ -15,6 +15,10 @@ const path = require('path');
 
 module.exports = {
   process(src, filename) {
+    if (filename.match(/\.svg$/)) {
+      return `module.exports = {ReactComponent: () => ('')};`;
```

Comment:
If we don’t want to render anything can this just return null?

Reply:
This is not quite accurate. There should be two exports: default and named one. Tests that expect this to render to the filename shouldn’t start failing.

Reply:
I wasn't exactly sure what to return there. Null would probably work. Also wasn't sure about the exports. How do you return a default and named export using `module.exports`?

Reply:
Maybe something like 
```
module.exports = {
  __esModule: true,
  default: name,
  ReactComponent: () => name
}
```